features/admin.policy-administration.v1/components/policy-administration-page-layout.tsx

```diff
@@ -189,7 +187,7 @@ const PolicyAdministrationPageLayout: FunctionComponent<PolicyAdministrationPage
 
     const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
         event.preventDefault();
-        setSubmittedSearchQuery(searchQuery);
+        setSubmittedSearchQuery(searchQuery.trim() !== "" ? `${searchQuery}*` : "*");
```

Comment:
Lets add to the beginning as well. So that will cover the case of `contains` as well